src/NUnitEngine/nunit.engine/Runners/ProcessRunner.cs

```diff
@@ -21,10 +21,10 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 // ***********************************************************************
 
-#if !NETSTANDARD1_6 && !NETSTANDARD2_0
+#if !NETSTANDARD // Dependency on RuntimeFrameworkService
```

Comment:
Thanks for adding the comment! I think we'll want to look at how we might be able to refactor to solve that maybe... (Outside of this PR, of course!)

Reply:
Ahh, target framework version. Making that bit .NET Standard friendly is gonna be fun! 😄

Reply:
Yes :D I'm willing to help with that!

In the latest force push, I added `// Dependency` comments on the files in the Agent folder, too.